File: dogmeet-projekt/dogMeet_laravel/app/Models/DogHistory.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class DogHistory extends Model
{
    use HasFactory;

    protected $table = "dog_history";
    public $timestamps = false;
    protected $fillable = [
        "breededWith_Type",
        "kidsBorn",
        "date",
        "dog_id"
    ];

    public function dog()
    {
        return $this->belongsTo(Dog::class);
    }
}

File: dogmeet-projekt-C#-PHP/dogMeet_admin/dogMeet_adminSite/dogMeet_adminSite/Model/DogHistory.cs
using System;

namespace dogMeet_adminSite.Model
{
    public class DogHistory
    {
        public int id { get; set; }
        public string breededWith_Type { get; set; }
        public int kidsBorn { get; set; }
        public DateTime date { get; set; }
        public int? dog_id { get; set; }

        public DogHistory() { }
    }
}
